explorer/README.md

```diff
@@ -0,0 +1,54 @@
+# Explorer
+
+Explorer website.
+
+## TOC
+
+- [Explorer](#explorer)
+  - [TOC](#toc)
+  - [Build system and dev environment](#build-system-and-dev-environment)
+  - [Environment variables](#environment-variables)
+    - [NPM scripts](#npm-scripts)
+  - [Running a local Rusk node](#running-a-local-rusk-node)
```

Comment:
Does this apply here?

Reply:
Didn't know that is why I left it there, being that that the explorer is in the rusk repo.